web-registry/src/components/organisms/ChooseCreditClass/ChooseClassTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+
+import OnBoardingSection from 'web-components/lib/components/section/OnBoardingSection';
+import ErrorBanner from 'web-components/lib/components/banner/ErrorBanner';
+import { JustifyContentValue } from 'web-components/lib/types/style/flex';
+import { Loading } from 'web-components/lib/components/loading';
+
+interface TemplateProps {
+  justifyContent?: JustifyContentValue;
```

Comment:
```suggestion
  justifyContent?: GridProps['justifyContent'];
```
and delete `/web-components/lib/types/styles/flex` - we shouldn't need to maintain that type

Reply:
It did already exist, sorry for that @mhagel

Reply:
thanks! done